techeria_controller/techeria_app/tests/test_views.py

```diff
@@ -3,4 +3,9 @@
 from techeria_app.models import *
 
 #https://adamj.eu/tech/2019/04/30/getting-a-django-application-to-100-percent-coverage/
-import json
\ No newline at end of file
+import json
+
+class productInfoTests(TestCase): # Needs to be fixed?
```

Comment:
I am not sure about this part! are you trying to fetch the info from the database?